Azure/azure-cli
# [App Service] Fix #14758: az webapp create errors when creating windows app with --runtime dotnetcore

**Description**  
Fixes #14758 
az webapp create fails when creating a windows app using a dotnetcore runtime.

**Testing Guide**  
- Create windows ASP:
`az appservice plan create -g RG -n Plan`
- Create dotnetcore webapp --runtime "DOTNETCORE|2.1" or "DOTNETCORE|3.1":
`az webapp create -g RG -p Plan -n Name --runtime "DOTNETCORE|3.1"`

Command fails with unexpected error: cli.azure.cli.core.util : 'configs'. Command should have worked successfully

---

This checklist is used to make sure that common guidelines for a pull request are followed.
